client/subchannel_test.go

```diff
@@ -0,0 +1,89 @@
+package client_test
+
+import (
+	"context"
+	"math/big"
+	"testing"
+
+	ethchanneltest "github.com/perun-network/perun-eth-backend/channel/test"
+	ethclienttest "github.com/perun-network/perun-eth-backend/client/test"
+	"perun.network/go-perun/apps/payment"
+	chtest "perun.network/go-perun/channel/test"
+	"perun.network/go-perun/client"
+	clienttest "perun.network/go-perun/client/test"
+	"perun.network/go-perun/wire"
+	pkgtest "polycry.pt/poly-go/test"
+)
+
+func TestSubChannelHappy(t *testing.T) {
+	rng := pkgtest.Prng(t)
+	const A, B = 0, 1 // Indices of clients.
+	s := ethchanneltest.NewSetup(t, rng, 2, ethclienttest.BlockInterval, TxFinalityDepth)
+	setups := ethclienttest.MakeRoleSetups(rng, s, [2]string{"Susie", "Tim"})
+	roles := [2]clienttest.Executer{
+		clienttest.NewSusie(t, setups[A]),
+		clienttest.NewTim(t, setups[B]),
+	}
+	// enable stages synchronization
+	stages := roles[A].EnableStages()
+	roles[B].SetStages(stages)
+
+	cfg := clienttest.NewSusieTimExecConfig(
```

Comment:
This is quite a large config, maybe use consts/variables for single parameters to make it more readable.

Reply:
Done